### DISCUSSION TITLE
No gcode  when using the Design Option.

### DISCUSSION THREAD

**Discussion by Geoffnut** [winder/Universal-G-Code-Sender#2603] [Help]
Hi all, I'm Geoff from the land downunder.
Firstly, thanks to everyone that has contributed to this program.
When using the design panel I can draw, write text and all  but alas there is no gcode generated from the designs.
I have pushed and prodded every button but nothing, also if I right click on the design the jog options work.
Beats me
Regards  Geoff

**Comment by breiler** [comment]
Hi Geoff, 

I made this video on how to get started with the designer. It is focused on laser engravers but it has the same workflow for mills.

https://youtu.be/Q8dBg5UuUPQ?feature=shared
